### PR TITLE
Project wizard improvements

### PR DESCRIPTION
**Pull request by waltersajtos**
## Description

While testing the wizard we ran into some small issues, this pull request addresses those issues.

## Type of change

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [ ] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

When adding a collaborator using the project wizard the role will always be "Developer":
1. Open project wizard
2. Add collaborator with a role not being "Developer". 
3. Go to next step. Go back to collaborator step.
4. Collaborator is now developer.

Source: "Add project manually" should be first option.
1. https://staging.dex.software/project/add
2. First option is disabled Gitlab source.

Update after delete project
1. Delete a project
2. Project still shows on the project overview until you refresh 

Icon on the first wizard page (GitLab & GitHub)
1. Open GitLab / GitHub wizard.
2. On the first step a placeholder is shown.

## Link to issue

Closes: #462
Closes: #460 
Closes: #492 
Closes: #490

### CODE REVIEW

**Review by RubenFricke — COMMENTED** [review]
Everything worked just as you mentioned. Good work Walter! The only remark I have is that the icon fix you made is not mentioned in the CHANGELOG.  

With this pull request, you would like to close #460, but I think you only tackled the visuals and not the interaction part? (I can still click 2 > 3 >4 and not 2 > 4) Maybe we can schedule a meeting to talk about this? 

In the created issue it was mentioned that the manual flow should be the first option. I don't understand the reason for this @niraymak. In my opinion, it makes more sense for it to be the last option because we want to stimulate users to import projects from external sources, thus the first options. If all these options are not applicable, we should offer the manual option as last resort. @liekevdvoort what are your thoughts on this?

File: src/app/modules/project/add/main/wizard/wizardPages/default/project-link/project-link.component.ts [hunk 1]
```diff
@@ -88,8 +88,9 @@ export class ProjectLinkComponent extends WizardStepBaseComponent implements OnI
    * Method to get the url of the icon of the project. This is retrieved
    * from the file retriever service
    */
-  public getIconUrl(project): SafeUrl {
-    return this.fileRetrieverService.getIconUrl(project.projectIcon);
+  public getSourceIconUrl(): SafeUrl {
+    console.log(this.selectedSource);
```
[the remark this answers is not in the document]
Comment (liekevdvoort) [reply src/app/modules/project/add/main/wizard/wizardPages/default/project-link/project-link.component.ts:92]: I don't know I think it should be logical for the first option to be the option that is always usable with or without a link. and if you do have a GitHub project I think you will recognize the logo way faster than the person without a supported import searching for the default option.

But I don't have a really strong opinion on it just my thoughts.

**Comment by waltersajtos** [comment]
I will include #492 in this PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,12 +11,20 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
+- Clicking to the next step in the wizard step-header will now skip the step & removed orange progress bar
```
Comment (niraymak) [inline CHANGELOG.md:14]:
```suggestion
- Clicking to the next step in the wizard step-header will now skip the step & removed orange progress bar - [#460](https://github.com/DigitalExcellence/dex-frontend/issues/460)
```

I think it became 2 lines by accident

File: CHANGELOG.md [hunk 3]
```diff
@@ -11,12 +11,20 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
...
+  - [#460](https://github.com/DigitalExcellence/dex-frontend/issues/460)
```
Comment (niraymak) [inline CHANGELOG.md:15]:
```suggestion
```
Not sure if this suggestion removes the line..

**Review by niraymak — CHANGES_REQUESTED** [review]
Great job! Works like a charm!

 Only found a mistake in the changelog. Will approve when commited 👍

**Review by niraymak — APPROVED** [review]

**Review by niraymak — APPROVED** [review]

**Review by RubenFricke — APPROVED** [review]
You tackled a lot of different things and it all worked flawlessly. Compliments 🥂 !

**Pull request merged**